index.html

```diff
@@ -186,31 +197,272 @@ <h4>
+          user agent.
+        </p>
+        <p>
+          The <dfn>home tab scope</dfn> is the set of URLs to which the home
+          tab context can be navigated to. All navigations within the
+          application to a URL within the home tab scope will be done in the
+          [=home tab context=], and all other navigations will be done in a
+          different [=application context=].
+        </p>
+        <p>
+          The [=home tab scope=] consists of the [=start URL=] and URLs that
+          match a pattern specified in the [=scope_patterns=] field.
+        </p>
+        <aside class="note">
+          Navigation to a URL within the [=home tab scope=] will open in the
+          home tab, leaving behind the tab that initiated the navigation.
+          Developers should ensure the tab left behind does not end up in an
+          unexpected state.
+        </aside>
+        <section>
+          <h4>
+            <code><dfn>scope_patterns</dfn></code> member
+          </h4>
+          <p>
+            The [=home_tab/scope_patterns=] member is a list of [=URL
+            pattern=]s that define the scope of the home tab.
           </p>
           <p>
-            The <dfn data-export="" data-lt="app-region">`app-region` </dfn>
-            property can be used to define with CSS which regions or elements in
-            for example a title bar are draggable.
+            If navigating from the home tab context to a [=URL=] |URL:url| that
+            is outside of the [=home tab scope=], the following steps are run:
           </p>
-          <ul>
-            <li>
-              To enable dragging an element, shall be set to `drag`.
+          <ol>
+            <li>Create a new [=application context=] in the current window.
             </li>
-            <li>
-              To disable dragging an element, shall be set to `no-drag`.
+            <li>Navigate it to |url|.
             </li>
-          </ul>
+          </ol>
+          <p>
+            If navigating to a [=URL=] |URL:url| within the [=home tab scope=],
+            the following steps are run:
+          </p>
+          <ol>
+            <li>Focus the [=home tab context=].
+            </li>
+            <li>Navigate it to |url|.
+            </li>
+          </ol>
+        </section>
+      </section>
+      <section>
+        <h3>
+          <code><dfn>new_tab_button</dfn></code> member
+        </h3>
+        <p>
+          The [=tab_strip/new_tab_button=] member is an ordered map that
+          describes the behaviour of a UI affordance (such as a button) which,
+          when clicked/activated, opens a new [=application context=] within
+          the application window. It has the following members:
+        </p>
+        <ul>
+          <li>[=new_tab_button/url=]
+          </li>
+        </ul>
+        <p>
+          If [=new_tab_button/url=] is within the [=home tab scope=], then the
+          new tab button may be hidden.
```

Comment:
Should 'may' be capitalized?

Reply:
Good call-out. I think it's actually good to strengthen this to a MUST.

And further, I think this rule should be moved to the parser: instead of simply saying "don't show the button in the UI", the parser should just ignore it if it's within scope. Would that change the behaviour at all? (I don't think so, although internally it would mean we are discarding the illegitimate new_tab URL rather than storing it in the app database, which is slightly risky if we want to change this rule later.)

Reply:
Yeh we could change the parser to ignore it.
Currently later in the code, if the new tab URL is unset, we use the start_url (which will then always be in the home tab scope so it gets hidden). But we could leave it unset, and make sure the parser sets it to start_url when relevant.
I don't know how likely it is to want to change this rule, seems like very strange behaviour if we allowed it.